tembo-cli/tests/tomls/multiple-instances/tembo.toml

```diff
@@ -0,0 +1,16 @@
+[defaults]
```

Comment:
this should be named something other than defaults, since defaults is supposed to mean defaults for all instances, not the name of an instance